src/report.tex

```diff
@@ -688,20 +688,22 @@ \subsubsection*{Auxilliary Codes}
 The auxiallary codes used in the this report are shown in Table 
 \ref{table:validation_aux}. These codes were not directly used to 
 produce results, but were used to either process or produce input
-and output data.
+and output data.  Moreover, because validation implies the comparison
+of physical models to experiments not all tools can be validated.  
+Instead, it is expected that such tools have bee verified through
+unit and regression testing.
 \begin{centering}
  \begin{table}[ht!]
   \begin{tabular}{c | c | c}
   \hline
-  Code & Validation Status & References \\    
+  Code & Validation/Verification Status & References \\    
   \hline
-  pyne.mesh & Validated by unit tests & - \\
-  pyne.r2s & Validated & \cite{pyne_r2s} \\
-  pyne.alara & Validated & \cite{pyne_r2s} \\
-  pyne.mcnp & Validated & \cite{pyne_r2s} \\
-  pyne.source\_sampling & Validated & \cite{pyne_r2s} \\
+  pyne.mesh & Verified by unit tests & n/a \\
+  pyne.r2s & Validated & \cite{biondoFED2016} \\
+  pyne.alara & Verified by unit tests & \cite{biondoFED2016}  \\
```

Comment:
@article{Biondo201677,
title = "Shutdown dose rate analysis with \{CAD\} geometry, Cartesian/tetrahedral mesh, and advanced variance reduction ",
journal = "Fusion Engineering and Design ",
volume = "106",
number = "",
pages = "77 - 84",
year = "2016",
note = "",
issn = "0920-3796",
doi = "http://dx.doi.org/10.1016/j.fusengdes.2016.03.004",
url = "http://www.sciencedirect.com/science/article/pii/S0920379616302009",
author = "Elliott D. Biondo and Andrew Davis and Paul P.H. Wilson",
keywords = "Shutdown dose rate",
keywords = "CAD",
keywords = "Tetrahedral mesh",
keywords = "Variance reduction "
}

bibtex entry for biondos article